gin/v8_platform.cc

```diff
@@ -359,7 +359,10 @@ class V8Platform::TracingControllerImpl : public v8::TracingController {
 // static
 V8Platform* V8Platform::Get() { return g_v8_platform.Pointer(); }
 
-V8Platform::V8Platform() : tracing_controller_(new TracingControllerImpl) {}
+V8Platform::V8Platform() : tracing_controller_(new TracingControllerImpl) {
+  // [RUN-1348] Allocate the time clamper non-divergently.
+  g_time_clamper.Get();
```

Comment:
This needs to be gated on IsRecordingOrReplaying() (and ideally a feature, it would be good if we had a generic feature for use with forcing eager initialization of things that are normally initialized lazily).

Reply:
I forgot the most important rule of changing chromium code again. I guess I don't do it often enough, so I have not internalized it - Will fix!